index.html

```diff
@@ -21,6 +21,50 @@
             <a class="back_con" href="#contact"">Contact</a>
         </div>
     </div>
+
+    <div id="display_work">
+        <div class="work_overlay">
+        </div>
+        <div class="center_card">
+            <div class="cardPop">
+                <div class="work_pop_title">
+                    <h4 id="work_pop_name"></h4>
+                    <a class="close_work">&times;</a>
+                </div>
+                <ul>
+                    <li class="worktime"><b>CANOPY</b></li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle">Back End Dev</li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle year"></li>
+                </ul>
+                <div>
+                    <img class="imageWork" alt="SnapshootPortfolio">
+                </div>
+                <div class="workInfoDesktop">
+                    <div>
+                        <p class="workDetail"></p>
+                    </div>
+                    <div class="techLiveSource">
+                        <ul class="client techBox">
+                            <li class="worktime tag tech1"></li>
+                            <li class="worktime tag tech2"></li>
+                            <li class="worktime tag tech3"></li>
+                        </ul>
+                        <div class="buttonBox">
+                            <a class="goLive">
+                                <button class="goButtons" type="button">See Live <img src="images/live.png" alt="Go live"></button>
+                            </a>
+                            <a class="goSource">
+                                <button class="goButtons" type="button">See Source <img src="images/source.png" alt="Go Source"> </button>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] In order to fulfill the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this milestone, kindly remove these HTML contents. The content for your pop-up modal and your projects in the work section should be created dynamically when the page loads. Meaning that you are to generate them using JavaScript by some means of DOM.

![image](https://user-images.githubusercontent.com/99715640/223842827-7a475dd0-cdee-486e-b0d1-b38dc6ff0e8b.png)